scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -98,6 +98,47 @@ task SplitIntervals {
   }
 }
 
+task CollectSequencingArtifactMetrics {
+    String gatk4_jar
+    File bam_file
+    String output_prepend
+    File ref_fasta
+    Boolean run_orientation_bias_filter
+
+    command {
+        if [ ${run_orientation_bias_filter} = true ]; \
```

Comment:
done, but do you know how to specify a workflow output that is only produced if a task in a conditional block is run?